README.md

```diff
@@ -10,9 +10,11 @@
 
 First, install all [build dependencies](docs/dev/dependencies.md).
 
-Clone this repository to `src/github.com/openshift/installer` in your [GOPATH](https://golang.org/cmd/go/#hdr-GOPATH_environment_variable). Then build the `openshift-install` binary with:
+Clone this repository to `src/github.com/openshift/installer` in your [GOPATH](https://golang.org/cmd/go/#hdr-GOPATH_environment_variable) and build the `openshift-install` binary with `hack/build.sh`, so:
 
 ```sh
+mkdir $GOPATH/src/github.com/openshift ; cd $GOPATH/src/github.com/openshift
```

Comment:
So building installer from source assumes basic go knowledge. and i think the text above is adequate to inform developer to clone the repo to correct location and run `hack/build.sh`

Reply:
so want me to abandon this? just checking.